docs/Container_Messages.rst

```diff
@@ -7,18 +7,23 @@ Once a Type has been registered with a Type message, Containers can begin using
 
 The body of a Container message consists of an array of objects with the following keywords:
 
-=============== =================================================================================================
-Name            Value
-=============== =================================================================================================
-``id``          Unique identifier of the Container.
-``typeid``        ID of the Type used by the Container.
-``typeversion`` Optional version of the Type used by the Container. If omitted, version 1.0.0.0 is used.
-``name`` 	    Optional friendly name for the Container.
-``description`` Optional description for the Container.
-``tags``        Optional array of strings to tag the Container.
-``metadata``    Optional key-value pairs associated with the Container.
-``indexes``     Optional array of Type Property ids to be used as secondary indexes for the Container.
-=============== =================================================================================================
+=====================  =================================================================================================
+Name                   Value
+=====================  =================================================================================================
+``id``                 Unique identifier of the Container.
+``typeid``             ID of the Type used by the Container.
+``action``             Optional: One of: ``create``, ``update``, or ``delete``. The value specified overrides the value of the ``action`` header. If omitted, the value of the ``action`` header is assumed. See :doc:`Headers`.
+``description``        Optional description for the Container.
+``indexes``            Optional array of Type Property ids to be used as secondary indexes for the Container.
+``metadata``           Optional key-value pairs associated with the Container.
+``name`` 	           Optional friendly name for the Container.
+``propertyoverrides``  Optional key-value pairs defining overrides to properties of the Type.
+``tags``               Optional array of strings to tag the Container.
+``typeversion``        Optional version of the Type used by the Container. The version must be of format x.x.x.x, where x must be a positive integer. If omitted, version 1.0.0.0 is used.
```

Comment:
I was imagining some type of error, but maybe we can see what other people think.

Reply:
I would think we should reject the message. We don't need to say bad request in the core OMF documentation because OMF is technically protocol independent, so a Bad Request would only be for Http